hypervisor/src/cpu.rs

```diff
@@ -10,18 +10,23 @@
 //
 //
 
+#[cfg(target_arch = "aarch64")]
+use std::sync::Arc;
+
```

Comment:
I don't see why this is needed.

Reply:
I needed that for `vm: &Arc<dyn crate::Vm>,`